docs/pages/repo/docs/handbook/tools/storybook.mdx

```diff
@@ -78,8 +78,7 @@ Next, we need to scaffold Storybook:
 
       ```shell
       cd workshop
-      pnpm dlx sb init --skip-install
-      pnpm install --save-dev @storybook/cli   # Manually install deps and CLI
```

Comment:
I specifically remember having to go through these steps in this order as the only way I could get it to work.

I think the Storybook team has released two majors since I wrote this doc, though, some some things may have changed. Can you confirm that you tested this manually?

Reply:
@anthonyshew yes I've got some screen shots just now to make sure once again that this works. Let me show you.

![image](https://github.com/vercel/turbo/assets/44067788/fd4cf153-9fea-4029-980a-53f4ec9c110d)


![image](https://github.com/vercel/turbo/assets/44067788/9a056d13-afe6-4a64-b319-c31c5db676d1)


![image](https://github.com/vercel/turbo/assets/44067788/cbe3e38a-6859-4137-b4cb-42c2ea8a1b7f)


![image](https://github.com/vercel/turbo/assets/44067788/d6d7f007-71b3-487b-808d-965fbdc10e44)


...


![image](https://github.com/vercel/turbo/assets/44067788/13257cfd-c036-4947-bceb-cb46744e6ce3)


![image](https://github.com/vercel/turbo/assets/44067788/b8a11e68-383b-42ca-8e8f-41ceadf27154)


Although this was just now with storybook version 8.0.0 .When I openeed this PR a couple of days ago the stable version of storybook was 7.6 and this also worked fine and adding storybook without installing dependencies was running into errors.

Reply:
Thank you for going the extra mile with the screenshots! 🙏

Let's add that to the docs in a separate PR, if you could!

Reply:
Thank you 🙏 

It's all done. Please let me know if you know better way to document it.